Fix clipping in webgl2_materials_texture3d

rlschuller opened the pull request:
**Description**

The camera position was too close to the 3D texture, causing clipping
from certain angles.

Before the repositioning:
![threejs_before](https://user-images.githubusercontent.com/11084889/136242637-a40e1225-a669-4d30-97f5-6ad0d5486466.png)

After the repositioning:
![threejs_after](https://user-images.githubusercontent.com/11084889/136242720-c8d59ea9-fdca-4761-a53c-78222ff62192.png)

rlschuller committed:
Fix clipping in webgl2_materials_texture3d

The camera position was too close to the 3D texture, causing clipping
from certain angles.

Mugen87 commented:
https://threejs.org/examples/webgl2_materials_texture3d
https://raw.githack.com/rlschuller/three.js/dev/examples/webgl2_materials_texture3d.html

TBH, I don't see how the PR improves the example. Do you mind explaining the mentioned clipping in more detail?

rlschuller committed:
Adjust frustrum and camera position

Move the object to the middle of the frustrum and make the depth of the
frustrum bigger.

WestLangley commented:
Make these temporary changes so you can see what is happening:
```js
scene.add( new THREE.AxesHelper( 256 ) );

const mesh = new THREE.Mesh( geometry, new THREE.MeshNormalMaterial() );
```

Fix the clipping by moving the camera back a bit;
```js
camera.position.set( - 64, - 64, 128 );
```

Then disable panning.
```js
controls.enablePan = false;
```

WestLangley commented:
I would do what I recommended. You can't prevent clipping when the user is able to pan without limitation.

rlschuller committed:
Disable pan and adjust camera

Disable pan, move camera to -64, -64, 128 and restore original frustum.

mrdoob commented:
Thanks!
